cron_tasks/check_schedule.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
Can I know why this scheduled job is here and not in aos-cd-jobs along with the other scheduled jobs? Unless there is a good reason, I would really like it to be in aos-cd-jobs

https://github.com/openshift-eng/aos-cd-jobs/tree/master/scheduled-jobs/scanning

Reply:
Different from the Jenkins job, this is a script that will build into the image. I used to think that moving this part to art-tools/art-cluster/pipelines, may be the right place along with task and job config. But consider we may set up a trigger to the image build when a new commit happens in the docker file repo. I think the art-tools repo updates more frequently than the art-dashboard-server repo and the script is more bonded with the server API so put it here. When the API is changed, it can be in the same PR and commit and the cluster only will rebuild once.

Reply:
From what I understand from the code, the new API endpoint serves the data from errata unmodified, so we do not need to redeploy art-dash-server as well. The purpose of the Dockerfile is only to run this script right? This can be achieved with Jenkins, by setting up a periodic job. That is also where similar jobs are running.

Reply:
So if I didn't miss anything, I don't see any need for a Dockerfile here, and we can achieve the result that you intended by separating out the `check_schedule.py` code to aos-cd-jobs

Reply:
true and false, the dockerfile only for run the script is true and we need to redeploy the server to provide service for the API.  I would like to try the cron job in art cluster instead of using the jenkins scheduled job. the art-dash-server plays a role has permission to connect errata and response filtered output to the requester, which makes the "job" side very light and shaped with efficiency without extra init step.